util/vpc-tools/db-clone.py

```
@@ -1,181 +1,250 @@
+                        help="RDS option group for the new instance.")
+    
+    parser.add_argument('-g', '--security-groups', nargs='+', dest='security_groups',
+                        help='ids one or more security groups that should be assigned to the new RDS instance')
+    
+    parser.add_argument('-x', '--sql-script', action='append', nargs=2,
+                        dest='sql_scripts', metavar=('DBNAME', 'SCRIPTFILE'),
+                        help=("runs SCRIPTFILE (a SQL script) against DBNAME on the cloned instance. "
+                        "This option can be specified multiple times for multiple scripts. "
+                        "NB: if you don't specify --password, scripts can only be run "
+                        "if the source instance has no password."))
+
+    parser.add_argument('-c', '--dns', dest='dns', 
+                        help="dns entry (CNAME) for the new instance (e.g. foo-bar.edx.org)")
+    
+    parser.add_argument('-f', '--db-file', dest='db_file',
+                        help=("path to YAML file used for create_dbs_and_users play. "
+                        "If not specified, no users/passwords other than the root one will be changed. "
+                        'Note that the "database_connection" section of the file will be ignored.'))
+
+    parser.add_argument('--skip-dbs', action='store_true', dest='ansible_skip_dbs',
+                        help=("don't run the part of the create_dbs_and_users play that creates dbs. "
+                        'Equivalent to "--skip-tags dbs"'))
+
+    parser.add_argument('--skip-users', action='store_true', dest='ansible_skip_users',
+                        help=("don't run the part of the create_dbs_and_users play that creates users. "
+                        'Equivalent to "--skip-tags users"'))
+    
+
+    args = parser.parse_args()
+    if not args.target_db:
+        args.target_db = 'from-{}-{:%Y-%m-%d-%H-%M-%S}'.format(args.source_db, datetime.now())
```

I haven't tested this case, I prefer letting the boto call fail noisily. Perhaps we should choose a naming convention less likely to overflow. @e0d you were interested in that, right?